src/Ozds.Client/Shared/MeterGraph.razor

```diff
@@ -175,17 +172,28 @@
   @* TIME STUFF *@
   private bool timerInitialized;
   private Timer timer = new();
-  private int updateMilisec = 10000;
+  private int updateMillisecond = 10000;
   private DateTimeOffset now = DateTimeOffset.UtcNow.LocalDateTime;
 
   protected override async Task OnInitializedAsync()
   {
+    _displayType = new List<(string type, string name)>
+    {
+      ("Voltage", T["Voltage"]), ("Current", T["Current"]), ("Active Power", T["Active Power"]), ("Reactive Power", T["Reactive Power"])
+    };
+
     aggregateQueries = ScopedServices.GetRequiredService<OzdsAggregateQueries>();
     measureQueries = ScopedServices.GetRequiredService<OzdsMeasurementQueries>();
     _meter = await ScopedServices
       .GetRequiredService<OzdsMeterTableQueries>()
       .GetMeterById(Source);
 
+
+    if (Source.StartsWith("schneider"))
```

Comment:
we should do this via capabilities
there is a capabilities namespace in Ozds.Business
please add `HasApparentPower` property to `ICapabilities`, add implementation for abb and schneider capabilities and use _meter.Capcabilities.HasApparentPower here